_includes/tutorials/naming-changelog-repartition-topics/kstreams/code/build.gradle

```diff
@@ -34,7 +34,7 @@ dependencies {
     implementation "org.apache.avro:avro:1.9.1"
     implementation "org.slf4j:slf4j-simple:1.7.30"
     implementation "org.apache.kafka:kafka-streams:2.4.1"
-    implementation "io.confluent:kafka-streams-avro-serde:5.4.1"
+    implementation "io.confluent:kafka-streams-avro-serde:5.5.1"
```

Comment:
This one def needs streams and test-utils to move to 2.5.1. 

Fails with 
```
517 | Exception in thread "main" java.lang.NoClassDefFoundError: org/apache/kafka/common/requests/IsolationLevel
518 | at org.apache.kafka.streams.StreamsConfig.<clinit>(StreamsConfig.java:770)
518 | at org.apache.kafka.streams.KafkaStreams.<init>(KafkaStreams.java:557)
520 | at io.confluent.developer.NamingChangelogAndRepartitionTopics.main(NamingChangelogAndRepartitionTopics.java:154)
521 | Caused by: java.lang.ClassNotFoundException: org.apache.kafka.common.requests.IsolationLevel
522 | at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
523 | at java.lang.ClassLoader.loadClass(ClassLoader.java:418)
523 | at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:352)
525 | at java.lang.ClassLoader.loadClass(ClassLoader.java:351)
```
There's a few more below this one, but I won't point them all out.